lib/screens/create_region_event_screens/interactor/viewmodels/create_region_event_bloc.dart

```diff
@@ -1,11 +1,14 @@
 import 'dart:io';
 import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
```

Comment:
is this needed in the bloc?

Reply:
I think that is due the import of DateTime and TimeOfDay in the Events and States, Not looks too bad.